drivers/watchdog/Kconfig.nrfx

```diff
@@ -10,3 +10,16 @@ config WDT_NRFX
 	select NRFX_WDT
 	help
 	  Enable support for nrfx WDT driver for nRF MCU series.
+
+config NRFX_WDT0
```

Comment:
Making Kconfig aware of instances is something we should not do anymore. It should be the sole role of DTS now. 

I know it is still present in many places, and that is required to be fixed. But let's not introduce new ones.

Could look at it?

Reply:
I agree, but this is consistent with what we do for all other nrfx based drivers. This is obviously to be corrected and I have it on my to-do list. But I think it will be better to do the clean up for all these drivers together, instead of introducing the new approach in this PR (this one is for getting nrfx 2.0.0 integrated).

Reply:
ok